influxdb/client.py

```diff
@@ -284,15 +285,18 @@ def write(self, data, params=None, expected_response_code=204,
         headers = self._headers
         headers['Content-type'] = 'application/octet-stream'
 
-        if params:
+        if params and 'precision' in params:
```

Comment:
We don't need this check here, this will default to None.

```
In [1]: params = {'one': 'two'}

In [2]: if params:
      :     precision = params.get('precision')
      : else:
      :     precision = None
      :

In [3]: precision

In [4]: params
Out[4]: {'one': 'two'}

In [5]: type(precision)
Out[5]: NoneType
```